Milestone: First Pull Request for Hello Microverse Project

Jord4N-AC opened the pull request:
In this first Milestone, I did the following:

- Created a new GitHub repo "hello-microverse".
- Created a new branch "new-feature" based on the main branch.
- Set up the linter for HTML and CSS.
- Added files (.gitignore, .html, .css, test.md).
- Added irrelevant files to .gitignore.
- Checked all the requirements.
- Opened a pull request.

Jord4N-AC committed:
add header to .html

Jord4N-AC committed:
add style to header

Jord4N-AC committed:
add 1 file to .gitignore

Jord4N-AC committed:
Add node_modules to .gitgi

Jord4N-AC committed:
update style.css

Jord4N-AC committed:
update index.html

Jord4N-AC committed:
check linters.yml

Jord4N-AC committed:
correct indentation erros style.css

Jord4N-AC committed:
Correct indentation erros style.css

Jord4N-AC committed:
edit .gitignore file

Jord4N-AC committed:
Update README.md

carlos-ssh commented on .gitignore:
```diff
@@ -0,0 +1,2 @@
+# .gitignore
```
- Kindly add your node_modules in your ``.gitignore``.

carlos-ssh commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
- Kindly fix your Linters locally, after that request another code review.

Jord4N-AC replied:
Hi @carlos-ssh!! 
Excuse me I'm a little confused about how to fix Linters locally, If you could provide more details about this, It would be awesome, thanks.

carlos-ssh commented on README.md:
```diff
@@ -1 +1,22 @@
+
+![projecVisual](https://user-images.githubusercontent.com/71242870/138964567-44e30695-3327-4b4b-b5f8-00cd3aea6223.PNG)
+
+
```
- Add instructions on how to install your project locally, and commands if it's needed.
- Add a License.

carlos-ssh commented on index.html:
```diff
@@ -0,0 +1,16 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="hello.css" type="text/css">
```
- Set the correct ``.css`` file.

carlos-ssh reviewed (changes requested):
Hi @Jord4N-AC , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._







![changesrequested](https://user-images.githubusercontent.com/22258274/138976507-6b8f0c1d-3932-4191-b0a8-82b18a30b99f.gif)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Jord4N-AC committed:
Add node_modules to .gitignore

Jord4N-AC committed:
Add .hintrc and .stylelintrc.json to .gitignore

Jord4N-AC committed:
Add package-lock.json and package.json to .gitignore

Jord4N-AC committed:
Add package-lock.json and package.json to .gitignore correctly

Jord4N-AC commented:
In this second attempt I:

- Added node_modules to my .gitignore and 
- Fixed my Linters locally and also request a new code review.

Thank you for your feedback.

elisha2kyakpo1 commented on README.md:
```diff
@@ -1 +1,22 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse
+This is my first project here in Microverse, doing this project, I could to apply all the new skills learnt.
+
+## Screenshot
+
+![projecVisual](https://user-images.githubusercontent.com/71242870/138964567-44e30695-3327-4b4b-b5f8-00cd3aea6223.PNG)
+
```
- [ ] The readme file is the first file to see or check if a user wants to know more about the project, in this case, you should add information about how a user who is new to software or version control(git) can get and use your project. For example, you can write something like this: To get this project on your local machine, run this command to clone the repo into your machine. run this command to get into the folder. Install code editor of your choice. Open this project in the code editor and run the project in the browser to see the project. checkout [this](https://github.com/microverseinc/readme-template) template for reference.

elisha2kyakpo1 commented on .gitignore:
```diff
@@ -0,0 +1,7 @@
+# .gitignore
+test.md
+node_modules
+.hintrc
+.stylelintrc.json
+package-lock.json
+package.json
```
- [ ] [OPTIONAL] The project instructions have highlighted what should be ignored. node_modules should always be ignored, for this project, the focus should be this 
![test](https://user-images.githubusercontent.com/53124548/139479138-63d40bec-5a39-4f12-a723-42b77fa23eee.png)

elisha2kyakpo1 reviewed (changes requested):
Hi @Jord4N-AC, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Jord4N-AC committed:
Update README.md

Jord4N-AC committed:
Modify to files on .gitignore

Jord4N-AC committed:
Edit .gitignore file

Jord4N-AC committed:
Set the correct .css file

Jord4N-AC committed:
Merge branch 'new-feature' of github.com:Jord4N-AC/hello-microverse into new-feature

Jord4N-AC commented:
I have successfully fixed the riquiered changes, including the optional one.
Thank you for reviewing my code.

fernandorpm reviewed (approved):
# :heavy_check_mark: Pull Request Approved! :heavy_check_mark:
WOW WHAT IS THIS PERFECTION??????? WERE HUMANS SUPPOSED TO BEHOLD SUCH POWER????

Congratulations on your hard work! Please remember to merge the project and don't forget that you're awesome! :heart:

<img src="https://i.imgur.com/gOjmIlL.gif" width="300px" />

- You have done a wonderful job on this project! To highlight:
  - you have demonstrated great skills in following the template :heavy_check_mark: 
  - you have paid attention to almost all of the details :heavy_check_mark: 
  - you have written a nice and clean code :heavy_check_mark: 
  - your PR is on point :ok_hand: :heavy_check_mark: 

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Just keep in mind that Microverse has a standard for the project requirements, to guarantee that you will be able to become a great developer, and both you and I have to follow them!

___
Cheers and Happy Coding! :heart:

> *As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*
